### PR TITLE
Formblock: Complete configuration form for user edit form

### PR DESCRIPTION
**Pull request by vacho**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
       "drupal/formblock": {
         "Feature: Node edit form block": "https://www.drupal.org/files/issues/2020-12-07/formblock-node_edit_form-3090910-5.patch",
         "Missing config schema for blocks": "https://www.drupal.org/files/issues/2020-12-30/missing_config_schema_blocks-3073274-2816415-3090910-4.patch",
-        "User edit form": "https://www.drupal.org/files/issues/2021-01-20/2915442-5.patch"
+        "User edit form": "https://git.drupalcode.org/issue/formblock-2915442/-/merge_requests/1/diffs.patch"
```
Comment (andypost) [inline composer.json:114]: it should work but could be broken anytime if MR will be updated

Comment (andypost) [reply composer.json:114]: OTOH it's totally ok

Comment (davidferlay) [reply composer.json:114]: Why not use patch of diff of commit ? 
https://git.drupalcode.org/issue/formblock-2915442/-/merge_requests/1/diffs.patch?commit_id=7cc09821cf69abeb2e0bb9dbe5cb21b8eba69ecc

If it works in composer, that's the way to go I guess

Comment (andypost) [reply composer.json:114]: ++ to pin on commit

Comment (vacho) [reply composer.json]: Updated to use a patch, such is most commonly used on drupal-composer projects. (also .diff should works)

Comment (davidferlay) [reply composer.json]: It's a shame to go back to old way using patch @vacho 
did you try using `https://git.drupalcode.org/issue/formblock-2915442/-/merge_requests/1/diffs.patch?commit_id=7cc09821cf69abeb2e0bb9dbe5cb21b8eba69ecc` in composer ?

Comment (vacho) [reply composer.json]: Yes david. this also works, and let work with gitlab MR more than patchs.

Comment (Sergey-Orlov) [reply composer.json]: current patch is not compatible with patch "Missing config schema for blocks"

**Pull request merged**